tests.py

```diff
@@ -1,11 +1,61 @@
 import data
 import unittest
+import sys
 
 
-class DataTest(unittest.TestCase):
-    @classmethod
-    def setUpClass(cls):
-        cls.data = data.DATA
+class BackportTestCase(unittest.TestCase):
```

Comment:
So the problem with something like this is that we are modifying the original class, this is mostly a no-no in OOP land, if we want to modify an existing class that we don't have control over the original definition we are better off creating a subclass and overwriting any definitions we want there. Make the OOP overlords happy :)